### PR TITLE
added flow

### PR DESCRIPTION
**Pull request by Teabue**
_Apparently, the task was pretty simple_ :^)

Implementation moves the object assumed at the mask center mass to the target point, perhaps in #1, it would be nice to display the center mass also?

@jojur, it will directly return a numpy array, so in #3 you can just convert it to tensor w. ` torch.tensor(flow).to(device)`, thereby you don't need to load the flow as they do in [generate.py](generate.py)

**Why not merged?**
The file location is temporary, it should be moved somewhere else @LenaRibena  was making some changes so we have src/vid_gen structure - so I'm not merging yet

### CODE REVIEW

**Comment by jojur** [comment]
I have integrated your (@Teabue) Python module into the very nice structure set up by (@LenaRibena ). Great job, both!

Merging now.
